src/command.rs

```diff
@@ -77,6 +81,40 @@ pub async fn init(kind: Option<PackageType>, name: Option<PackageName>) -> miett
     Ok(())
 }
 
+/// Initializes a project with the given name in the current directory
+pub async fn new(kind: Option<PackageType>, name: PackageName) -> miette::Result<()> {
+    let mut package_dir: PathBuf = ".".into();
+    package_dir.push::<String>(name.clone().into());
```

Comment:
What is the rationale against just constructing a pathbuf directly from the package name?

Reply:
Hmm, yeah nothing really, I'll change it